dependencies/pom.xml

```diff
@@ -63,7 +63,7 @@
         <version.lib.h2>2.2.220</version.lib.h2>
         <version.lib.hamcrest>1.3</version.lib.hamcrest>
         <version.lib.handlebars>4.3.1</version.lib.handlebars>
-        <version.lib.hibernate>6.1.7.Final</version.lib.hibernate>
+        <version.lib.hibernate>6.3.1.Final</version.lib.hibernate>
```

Comment:
After this upgrade we should no longer need to force upgrade `byte-buddy`. Could you verify that and then remove version management of byte-buddy from this file as well as top level `pom.xml` and `applications/mp/pom.xml`

Reply:
Done